src/components/LoginForm/i18n.js

```diff
@@ -0,0 +1,6 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'LoginForm', {
+  labelEmail: 'Email',
+  labelPassword: 'Password'
+});
```

Comment:
In that constant/i18n.js I declared global things like FormValidations and Errors not fields

Reply:
I think it's fine that this is here, I don0t think Password and Email are global, at least in this app. In another it might be reusable, and in that case I would probably move it to the constants file alejo mentions